src/App/component/ComparisonPage.jsx

```diff
@@ -72,18 +72,34 @@ function ComparisonPage(prop) {
   };
 
   const projectId = localStorage.getItem("projectId")
-  const jwtToken = localStorage.getItem("jwtToken")
+//   const jwtToken = localStorage.getItem("jwtToken")
```

Comment:
Confusing why commented this line? Don't you need the jwt anymore??

Reply:
![error](https://user-images.githubusercontent.com/40756699/147306848-b3bcce9a-d1e9-4be1-a903-e7686019489f.JPG)

It shows that I indeed haven't use it since I adjusted all API-related blocks according to the suggestion above.